earth_enterprise/src/installer/install_server.sh

```diff
@@ -18,41 +18,49 @@
 
 set +x
 
-# script arguments
+. common.sh
```

Comment:
This only works if you are within the installer directory. I expected it to work from at least the `src` direcctory (one level up). Ideally work it out, but at least note it.